src/extension.ts

```diff
@@ -203,10 +203,18 @@ export async function activate(context: vscode.ExtensionContext) {
 					const decoded = codec.decode(arg0);
 
 					if (decoded._tag === 'Right') {
-						await sourceControl.createIssue(decoded.right);
-						vscode.window.showInformationMessage(
-							'Successfully created issue.',
+						const { html_url } = await sourceControl.createIssue(
+							decoded.right,
 						);
+						const messageSelection =
+							await vscode.window.showInformationMessage(
+								'Successfully created issue',
```

Comment:
do you want to show the link URL here as well?

Reply:
I think i will be nice to have it